components/ResultsPage/ResultsBoxes.tsx

```diff
@@ -0,0 +1,208 @@
+import { observer } from 'mobx-react'
+import { useRouter } from 'next/router'
+import Image from 'next/image'
+import React from 'react'
+import { numberToStringCurrency } from '../../i18n/api'
+import { WebTranslations } from '../../i18n/web'
+import { Locale } from '../../utils/api/definitions/enums'
+import { useStore, useTranslation } from '../Hooks'
+import { ResultsTableRow } from './ResultsTableRow'
+import { MessageBox } from './MessageBox'
+
+export const ResultsBoxes = observer(() => {
+  const root = useStore()
+  const tsln = useTranslation<WebTranslations>()
+  const currentLocale = useRouter().locale
+
+  const locale = currentLocale == 'en' ? Locale.EN : Locale.FR
+
+  // Send the details and eligibility results separately and create a new column
+  return (
+    <div>
+
+      {/* Your may be eligible */}
+
+      <h2 id="eligible" className="h2 mt-5"><Image src="/eligible.png" alt="Eligible" width={30} height={30} />&nbsp; {tsln.resultsPage.youMayBeEligible}</h2>
+
+      <div className='pl-12'>
+        {tsln.resultsPage.basedOnYourInfo}
+
+        <ul className="pl-5 list-disc text-content font-semibold">
+          {root.oas?.eligibility?.detail.split('\n')[0] === ('Eligible' || 'Admissible') ? <li key={root.oas}>{tsln.oas}</li> : ''}
```

Comment:
you should be using the `result` property rather than splitting `detail` and checking for an english/french string. See here for what that property is: https://github.com/DTS-STN/eligibility-estimator/blob/develop/client-state/store.ts#L24